File: api/game/Log.py


class Log:
	def __init__(self):
		self.log = {}

	def addLog(self, example, answer, result, time, difficulty):
		self.log[len(self.log) + 1 if len(self.log) != 0 else 1] = {
			"example": example,
			"answer": answer,
			"result": result,
			"time": time,
			"difficulty": difficulty
		}

	def getLog(self):
		return self.log

File: src/game/Log.js


export class Log {
    constructor() {
        // {id: 1, example: [5, 5], correct: true, result: 10, time: 10, difficulty: 1}
        this.log = {};

    }

    addLog(example, correct, result, time, difficulty) {
        this.log[Object.keys(this.log).length + 1] = {
            example: example,
            correct: correct,
            result: result,
            time: time,
            difficulty: difficulty
        };
    }

    getLog() {
        return this.log;
    }


}